webanno/webanno
# Feature/1775 curation cas creation

Merge cases for the curation and correction modes should not be (re-)initialized on error, instead when no previous cas was saved.

**What's in the PR**
* check for existing cas when getting merge cas
* ~refactor: moved common functionalities of `CorrectionDocumentService` and `CurationDocumentService` to `MergeDocumentService`~

**How to test manually**
* Check that curation still works
* Check that correction/automation still works
